gui/dump-all-dialogs.cpp

```diff
@@ -155,17 +148,37 @@ void dumpAllDialogs(const Common::String &message) {
 	int original_window_width = ConfMan.getInt("last_window_width", Common::ConfigManager::kApplicationDomain);
 	int original_window_height = ConfMan.getInt("last_window_height", Common::ConfigManager::kApplicationDomain);
 	Common::List<Common::String> list = Common::getLanguageList();
-
+	const int res[] = {320, 200,
+					   320, 240,
+					   640, 400,
+					   640, 480,
+					   800, 600,
+					   0};
+
+	ConfMan.setBool("force_resize", true, Common::ConfigManager::kApplicationDomain);
 	Common::FSNode dumpDir("snapshots");
+
 	if (!dumpDir.isDirectory())
 		dumpDir.createDirectory();
 
-	// Iterate through all languages available
-	for (Common::String lang : list) {
-		// Iterating through the resolutions doesn't work so you have to manually change it here.
-		// TODO: Find a way to iterate through the resolutions using code.
-		Common::Array<int> res_to_test = {800, 640, 320};
-		dumpDialogs(message, res_to_test[0], lang);
+	// Iterate through all resolutions available
+	for (const int *r = res; *r; r += 2) {
+		int w = r[0];
+		int h = r[1];
+
+		// Update resolution
+		ConfMan.setInt("last_window_width", w, Common::ConfigManager::kApplicationDomain);
```

Comment:
Align this vertically, please, e.g. add one more space after `width"`, so `w` is right above `h`